package com.xquare.domain.meal.exception

internal class NetworkMealNotFoundException : RuntimeException("Meal from network not found.")
